File: Web/Prakhar-08_PrakharRastogi_1923it1011_3/GettingStarted/Digits of a Number.cpp
#include <iostream>
#include <bits/stdc++.h>

using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;

    //write your code here
    string s = to_string(n);
    for(int i=0; i<s.size(); i++) {
        int a = s[i] - '0';
        cout << a << endl;
    }
    
    return 0;
}

File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/DigitsOfANumber.java
import java.util.*;
    
    public class Main{
    
    public static void main(String[] args) {
      // write your code here  
      Scanner sc=new Scanner(System.in);
      int n=sc.nextInt();
      String s=Integer.toString(n);
      for(int x=0;x<s.length();x++){
          System.out.println(s.charAt(x));
      }
     }
    }